sycl/source/detail/device_info.hpp

```diff
@@ -691,6 +705,66 @@ struct get_device_info_impl<
   }
 };
 
+template <>
+struct get_device_info_impl<
+    std::vector<ext::oneapi::experimental::matrix::combination>,
+    ext::oneapi::experimental::info::device::matrix_combinations> {
+  static std::vector<ext::oneapi::experimental::matrix::combination>
+  get(const DeviceImplPtr &Dev) {
+    using namespace ext::oneapi::experimental::matrix;
+    using namespace ext::oneapi::experimental;
+    architecture DeviceArch = get_device_info_impl<
+        ext::oneapi::experimental::architecture,
+        ext::oneapi::experimental::info::device::architecture>::get(Dev);
+    if (architecture::intel_cpu_spr == DeviceArch)
+      return {
+          {16, 16, 64, 0, 0, 0, matrix_type::uint8, matrix_type::uint8,
+           matrix_type::sint32, matrix_type::sint32},
+          {16, 16, 64, 0, 0, 0, matrix_type::uint8, matrix_type::sint8,
+           matrix_type::sint32, matrix_type::sint32},
+          {16, 16, 64, 0, 0, 0, matrix_type::sint8, matrix_type::uint8,
+           matrix_type::sint32, matrix_type::sint32},
+          {16, 16, 64, 0, 0, 0, matrix_type::sint8, matrix_type::sint8,
+           matrix_type::sint32, matrix_type::sint32},
+          {16, 16, 32, 0, 0, 0, matrix_type::bf16, matrix_type::bf16,
+           matrix_type::fp32, matrix_type::fp32},
+      };
+    else if (architecture::intel_gpu_cfl == DeviceArch)
```

Comment:
I don't think there XMX (DPAS) unit in intel_gpu_cfl. If true, we should remove this.
Also, I don't see combinations for intel_gpu_pvc.

Reply:
Oh, my bad, was used for debugging purposes. Should be intel_gpu_pvc here